src/helper_ackPayload.py

```diff
@@ -1,22 +1,27 @@
-"""This module is for generating ack payload."""
+"""
+This module is for generating ack payload
+"""
 
-import highlevelcrypto
-import helper_random
 from binascii import hexlify
 from struct import pack
-from addresses import encodeVarint
 
-# This function generates payload objects for message acknowledgements
-# Several stealth levels are available depending on the privacy needs;
-# a higher level means better stealth, but also higher cost (size+POW)
-#   - level 0: a random 32-byte sequence with a message header appended
-#   - level 1: a getpubkey request for a (random) dummy key hash
-#   - level 2: a standard message, encrypted to a random pubkey
+import highlevelcrypto
+import helper_random
+from addresses import encodeVarint
 
 
 def genAckPayload(streamNumber=1, stealthLevel=0):
-    """Generate and return payload obj."""
-    if (stealthLevel == 2):      # Generate privacy-enhanced payload
+    """
+    Generate and return payload obj.
+
+    This function generates payload objects for message acknowledgements
+    Several stealth levels are available depending on the privacy needs;
+    a higher level means better stealth, but also higher cost (size+POW)
+       - level 0: a random 32-byte sequence with a message header appended
```

Comment:
WARNING: Unexpected indentation.

there should be a blank line before levels list